distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -281,10 +281,24 @@ public void testReadDerivedStorage() throws IOException {
   @Test
   public void testMetacardTransformersFromRest() {
     String id = ingestGeoJson(getFileContent(JSON_RECORD_RESOURCE_PATH + "/SimpleGeoJsonRecord"));
+    String title = "myTitle";
+    String geoCoordinates = "30.0 10.0";
+
+    String idXPath = "/metacard[@id='" + id + "']";
+    String titleXPath = "/metacard/string[@name='title'][value='" + title + "']";
+    String geoXPath = "/metacard/geometry";
+    String geoCoordsXPath = geoXPath + "//Point[pos='" + geoCoordinates + "']";
 
     String url = REST_PATH.getUrl() + id;
     LOGGER.info("Getting response to {}", url);
-    when().get(url).then().log().all().assertThat().body(hasXPath("/metacard[@id='" + id + "']"));
+
+    // tests that metacard properties are properly set
+    when().get(url).then().log().all().assertThat().body(hasXPath(idXPath));
+    when().get(url).then().log().all().assertThat().body(hasXPath(titleXPath));
+
+    // tests that the geometry coordinates are properly set
+    when().get(url).then().log().all().assertThat().body(hasXPath(geoXPath));
+    when().get(url).then().log().all().assertThat().body(hasXPath(geoCoordsXPath));
```

Comment:
❗️ This test is not testing the new functionality of the `GeoJsonInputTransformer` from: https://github.com/codice/ddf/pull/4992
The "metadata" field on the `SimpleGeoJsonRecord` is too simple, and the output of it is not being checked in this test.

Reply:
@brjeter Can I create a new JSON record file with more complex metadata? Is there anything else I would need to do for that like adding to documentation somewhere?

Reply:
Yes you can just create a new records in the itest resources. In the same place as the `SimpleGeoJsonRecord`.